cmd/skupper/skupper.go

```diff
@@ -534,7 +534,7 @@ var exposeOpts ExposeOptions
 func NewCmdExpose(skupperCli SkupperServiceClient) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:    "expose",
-		Short:  "Expose a set of pods through a Skupper address",
```

Comment:
Not too sure about this as it affects both podman and k8s cli

Reply:
I'd recommend updating the `Short` description as part of the `ExposeFlags()` and `UnexposeFlags()`
within `skupper_podman_service.go`. This way it'd only affect podman sites.